source/ui/BioXAS/BioXASXASScanConfigurationView.cpp

```diff
@@ -20,8 +20,8 @@ const AMScanConfiguration* BioXASXASScanConfigurationView::configuration() const
 
 void BioXASXASScanConfigurationView::setConfigurationName(BioXASXASScanConfiguration *configuration, const QString &newName)
 {
-	if (configuration && configuration->userScanName() != newName)
-		configuration->setUserScanName(newName);
+	if (configuration)
```

Comment:
Is there a reason why the second part of the check was removed along with the change from using name() instead of userScanName()?